consensus/hotstuff/voter/voter_test.go

```diff
@@ -83,7 +84,7 @@ func testBelowVote(t *testing.T) {
 
 	_, err := voter.ProduceVoteIfVotable(block, curView)
 	require.Error(t, err)
-	require.Contains(t, err.Error(), "not for current view")
+	require.Contains(t, err.Error(), "expecting block for current view")
```

Comment:
we have changed what error is returned, let's make sure that we are getting an exception and not a sentinel error